### ISSUE TITLE
Attempt_Catch may use Master ball or Great ball even if it't not present in inventory

### ISSUE THREAD

**Issue by joychugh** [j-e-k/pgoapi#18]
```
    def attempt_catch(self, encounter_id, spawn_point_guid):
        for i in range(1,4):
            r = self.catch_pokemon(
                normalized_reticle_size= 1.950,
                pokeball = i,
                spin_modifier= 0.850,
                hit_pokemon=True,
                normalized_hit_position=1,
                encounter_id=encounter_id,
                spawn_point_guid=spawn_point_guid,
                ).call()['responses']['CATCH_POKEMON']
            if "status" in r:
                return r

```
1 - normal
2- great
3 - ultra
4 - master

This behavior may result in softban.

I can attempt to fix it.

**Comment by j-e-k** [comment]
I wasn't aware this would cause a softban, a bix would be appreciated.

**Comment by wesselvdv** [comment]
This will also crash the application. If you have no poke balls, it wil continue to attempt a catch and crash.

**Comment by j-e-k** [comment]
@joychugh Can you take over this? Let me know if you need help

**Comment by joychugh** [comment]
@j-e-k i almost have a solution ill push it up

**Comment by j-e-k** [comment]
@joychugh Awesome, thanks again for the contributions!

**Comment by j-e-k** [comment]
Closed by #23
